modules/custom/openy_system/openy_system.links.task.yml

```diff
@@ -0,0 +1,5 @@
+#system.openy_modules_list:
```

Comment:
remove commented code, please

Reply:
@podarok we will uncomment it right after merging since we will use it further. I do not think this is a blocker.